src/Farmer/Arm/Sql.fs

```diff
@@ -0,0 +1,77 @@
+[<AutoOpen>]
+module Farmer.Arm.Sql
+
+open Farmer
+
+type Server =
```

Comment:
I noticed this with the Postgres implementation but with AutoOpen and using a generic name like `Server` we are going to run into ambiguity issues. At some stage soon we may want to change this to `SqlServer`. Doesn't have to be this PR though.

Reply:
Mmmm. Perhaps we should turn AutoOpen off for the Arm ones. Not sure. I wanted to keep the names identical to the official names so it aligned with the ARM terminology (including namespaces). Perhaps we will need to rethink this.